composer.json

```diff
@@ -15,6 +15,8 @@
         "php-http/httplug": "^2.0",
         "php-http/message-factory": "^1.0",
         "php-http/message": "^1.6",
+        "psr/http-factory": "^1.0",
+        "psr/http-message": "^1.0",
```

Comment:
why do you add the psr message standard? i don't see any changes in regard to that package?

Reply:
Strictly speaking, this was not related to this PR, but I included it for the same reason I included the factory: it is directly used by the codebase. I didn't create a separate PR purely because there would be merge conflicts.

Reply:
okay. both are sort of an "optional dependency", but i don't see any harm in pulling in the PSRs. fine for me